tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -626,6 +626,12 @@ def fit(self,
                 See the `ag_args_ensemble` argument from "Advanced functionality: Custom AutoGluon model arguments" in the `hyperparameters` argument documentation for valid values.
                 Identical to specifying `ag_args_ensemble` parameter for all models in `hyperparameters`.
                 If a key in `ag_args_ensemble` is already specified for a model in `hyperparameters`, it will not be altered through this argument.
+            included_model_types : list, default = None
+                To only include listed model for training during `fit()`.
+                Models that are listed in `included_model_types` but not in `hyperparameters` will be ignored.
+                Reference `hyperparameters` documentation for what models correspond to each value.
+                Useful when a only a subsample of model needs to be trained and the `hyperparameters` dictionary is difficult or time-consuming.
+                    Example: To include both 'KNN' and 'custom' models, specify `included_model_types=['KNN', 'custom']`.
```

Comment:
nit: Switch example to `['GBM', 'FASTAI']`, which is a more realistic setting a user would pick